src/libawkward/array/Record.cpp

```diff
@@ -197,7 +197,7 @@ namespace awkward {
   }
 
   const std::shared_ptr<Content> Record::flatten(int64_t axis) const {
-    throw std::runtime_error("FIXME: not implemented");
+    return array_.flatten(axis);
```

Comment:
Flatten on a single scalar record should be an error. Something like, "Record is not an array".